### PR TITLE
add Mega Menus to Pro Features

### LINKED ISSUE #5: Add Mega Menu card to /wp-admin/admin.php?page=crio_pro_features
Copy: Create custom WYSIWYG submenus with images, text, links, and more.

Icon: welcome-widgets-menus

Move the 'new' banner to this new card. recommend removing White Labeling or Blog Customization.

### PR DESCRIPTION
**Pull request by jamesros161**
Resolves #5

Testing Instructions: 

[crio-2.19.0-issue-5.zip](https://github.com/BoldGrid/crio/files/10450225/crio-2.19.0-issue-5.zip)

1. Activate theme without Crio Pro installed.
2. Ensure the number '1' is shown under Crio > Pro Features.
3. View the Pro Features page, and ensure the first item shown is the 'Mega Menu' Card and that it has the 'New' banner.
4. Leave the page, or refresh, and ensure the 'New' banner disappears and that the '1' is no longer shown under Crio > Pro Features.

### CODE REVIEW

**Review by avonville — APPROVED** [review]

File: src/includes/configs/pro-feature-cards.config.php [hunk 1]
```diff
@@ -9,29 +9,29 @@
  */
 
 return array(
-	'mega-menus'      => array(
+	'mega-menus'          => array(
```
Comment (avonville) [inline src/includes/configs/pro-feature-cards.config.php:12]: spacing is off for =>

**Review by avonville — CHANGES_REQUESTED** [review]

**Comment by nicolepaschen** [comment]
looks good ty!

**Comment by nicolepaschen** [comment]
beta1 looks good

**Comment by nicolepaschen** [comment]
RC1 looks good

**Pull request merged**